jmclient/jmclient/taker_utils.py

```diff
@@ -111,15 +112,15 @@ def direct_send(wallet_service, amount, mixdepth, destination, answeryes=False,
                 + "\n\nWARNING: This transaction if broadcasted will PERMANENTLY DESTROY your bitcoins\n"
         else:
             #regular sweep (non-burn)
-            fee_est = estimate_tx_fee(len(utxos), 1, txtype=txtype)
+            fee_est = estimate_tx_fee(len(utxos), 1, txtype=txtype, outtype=outtype)
             outs = [{"address": destination, "value": total_inputs_val - fee_est}]
     else:
         #not doing a sweep; we will have change
         #8 inputs to be conservative
-        initial_fee_est = estimate_tx_fee(8,2, txtype=txtype)
+        initial_fee_est = estimate_tx_fee(8,2, txtype=txtype, outtype=outtype)
```

Comment:
In case it wasn't obvious, the rather weird "let's try 8 and see what happens" is, of course, for the purpose of selecting utxos. This first step is a guess anyway, so I'm not sure it matters at all here.

Reply:
Yes. See https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/921/files#r701344743 for context.